libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -1,7 +1,7 @@
 <ng-container *ngIf="(readingList$ | async).length > 0; else empty">
   <div class="reading-list-item" *ngFor="let b of readingList$ | async">
     <div>
-      <img class="reading-list-item--cover" [src]="b.coverUrl" />
+      <img alt="{{b.title}}" class="reading-list-item--cover" [src]="b.coverUrl" />
```

Comment:
Is this appropriate? Now we have two elements which would be read out as `b.title` - would that be ok for a screen reader?

Reply:
I append the cover after the book title so that the screen reader can easily identify as cover image.